frontend_build/src/webpack.config.base.js

```diff
@@ -80,6 +80,8 @@ var config = {
       'kolibri_module': path.join(__dirname, '..', '..', 'kolibri', 'core', 'assets', 'src', 'kolibri_module'),
       'core-base': path.join(__dirname, '..', '..', 'kolibri', 'core', 'assets', 'src', 'core-base'),
       'core-theme.styl': path.join(__dirname, '..', '..', 'kolibri', 'core', 'assets', 'src', 'core-theme.styl'),
+      'content-renderer': path.resolve('kolibri/core/assets/src/content-renderer'),
+      'content_renderer_module': path.resolve('kolibri/core/assets/src/content_renderer_module'),
```

Comment:
at some point we need to settle on either underscores, dashes, or camelcase in file names. topic for later.